pkg/pmem-device-manager/pmd-util.go

```diff
@@ -2,13 +2,16 @@ package pmdmanager
 
 import (
 	"fmt"
+	"sync"
 
 	pmemexec "github.com/intel/pmem-csi/pkg/pmem-exec"
 	"k8s.io/klog/glog"
 	"os"
 	"strconv"
 )
 
+var devicemutex = &sync.Mutex{}
```

Comment:
Please add an explanation what is protected by this mutex, i.e. which operations must be done while holding the mutex.

Reply:
added a comment, force-pushed